src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -61,14 +61,21 @@ const styles = (theme: Theme): StyleRules =>
     //   justifyContent: 'flex-end',
     // },
     sectionTitle: {
-      color: theme.palette.text.secondary,
+      textAlign: 'center',
+      paddingTop: 0,
+      paddingBottom: 0,
+    },
+    sectionTitleText: {
+      color: theme.palette.text.primary,
```

Comment:
Note the change of color for the section header from secondary to primary. This improves readability.